packages/web/localizations/en.json

```diff
@@ -168,7 +168,7 @@
       "waitTime": "20 seconds",
       "warningLossFunds": "Warning: Withdrawing to central exchange address will result in loss of funds."
     },
-    "myPools": "My Pools",
+    "myPools": "Your Pools",
```

Comment:
fyi this change will only be reflected in english language, not foreign languages. Run `yarn machine-translate` in the root to get translations auto generated

Reply:
There's an issue with machine translate and inlang is fixing it ASAP

Reply:
Yeah.  It wasn't working for me.  I think we just run one big `yarn machine-translate` before deployment